modules/ROOT/pages/apikit-odatav4-migrating-api.adoc

```diff
@@ -0,0 +1,14 @@
+== Transforming a RAML OData v2 API to a CSDL OData v4 API
+ifndef::env-site,env-github[]
+include::_attributes.adoc[]
+endif::[]
+
+If you were using the previous APIkit OData extension, this new APIkit feature enables you to reuse the type definition of an OData v2 API to scaffold an OData v4 API by transforming an `odata.raml` file to an `odata.csdl.xml` file automatically.
```

Comment:
Flip to put the user task first. (Avoid "feature enables you to" construction)

Reply:
OK! Thanks!